src/views/RecIM/components/List/Listing/index.jsx

```diff
@@ -409,12 +424,29 @@ const ParticipantListing = ({
     handleClose();
   };
 
-  const handleRemoveFromTeam = async () => {
+  const reinstateMember = async () => {
+    let editedParticipant = {
+      Username: participant.Username,
+      RoleTypeID: 3,
+    }; // Role 3 is member
+    await editTeamParticipant(parseInt(teamIDParam), editedParticipant);
+    handleClose();
+  };
+
+  const handleMakeInactive = async () => {
     let editedParticipant = {
       Username: participant.Username,
       RoleTypeID: 6,
     }; // Role 6 is inactive
+    await editTeamParticipant(parseInt(teamIDParam), editedParticipant);
```

Comment:
was the parse int necessary here? would it work without?

Reply:
I'm not sure if it's necessary as I haven't tried without it. It's based on the boiler plate set up with the old promote logic from when the horizon menu was first made.